js/game.js

```diff
@@ -9,8 +11,80 @@ function whack(rat) {
   rat.classList.add('hidden')
 }
 
+/**
+ * Hide all the rats.
+ *
+ * @param {Nodelist} rats All the rats.
+ */
+function hideRats(rats) {
+    rats.forEach(function(rat) {
+        rat.classList.add('hidden')
+    })
+}
+
 rats.forEach(function(rat) {
   rat.addEventListener('click', function() {
     whack(rat)
   })
 })
+
+/**
+ * this function randomises between the times entered
+ * @param times (amount of time)
+ * @returns one of the times entered at random
+ */
+function random_time(times) {
+  return times[Math.floor(Math.random()*times.length)];
+}
+
+/**
+ * function selects random mole displays it and hides it after a defined period of time
+ */
+function showRat(){
+    var rats = document.querySelectorAll('.rat.hidden')
+    if (rats.length > 0 ) {
+        var randomRat = rats[Math.floor(Math.random() * rats.length)]
+        randomRat.classList.remove('hidden')
+        var time = [3000, 2000, 1000];
+
+        setTimeout(function () {
+            randomRat.classList.add('hidden')
+        }, random_time(time))
+    }
+    gameLoop()
+}
+
+/**
+ * keeps track of time
+ * @returns increments of 1 to a value
+ */
+function gameClock() {
+    gameTimer = gameTimer + 1
+    return gameTimer
```

Comment:
you dont need to return here? your editing a global variable so return does nothing